source/draw/RasterPacked.ooc

```diff
@@ -70,7 +70,7 @@ RasterPackedCanvas: abstract class extends RasterCanvas {
 		(sourceWidth, sourceHeight) := (sourceBox size x, sourceBox size y)
 		(sourceStartColumn, sourceStartRow) := (sourceBox leftTop x, sourceBox leftTop y)
 		(resultStartColumn, resultStartRow) := (resultBox leftTop x, resultBox leftTop y)
-		(sourceStride, resultStride) := (source stride / bytesPerPixel, target stride / bytesPerPixel)
```

Comment:
We didn't notice it before as bilinear resampling was tested only on monochrome images.